src/components/badge/badge.twig

```diff
@@ -0,0 +1,22 @@
+{% set _extra_content %}
+  {% block extra %}
+    {% if icon %}
+      {% include '@cloudfour/components/icon/icon.twig' with {
+        name: icon
+      } only %}
```

Comment:
I noticed the Firefox a11y inspector was complaining about this being an unlabeled image:

<img width="246" alt="Screen Shot 2021-07-12 at 9 33 25 AM" src="https://user-images.githubusercontent.com/5798536/125324055-5d4a9a80-e2f4-11eb-83ee-7f01569b4839.png">

Since this is decorative should we [add `aria-hidden="true"`](https://css-tricks.com/accessible-svgs/#example-2-standalone-icon-decorative) or `role="presentation"`?

(FWIW this does read correctly in VoiceOver, but I wonder if it could lead to issues in other assistive technologies.)

Reply:
D'oh! This is an oversight on my part, thank you for pointing it out.